lib/src/new-input-text/NewInputText.jsx

```diff
@@ -318,22 +316,25 @@ const DxcNewInputText = React.forwardRef(
 
     const incrementNumber = () => {
       const numberValue = value || innerValue;
```

Comment:
Wouldn't it be better: `const numberValue = value ?? innerValue;` ? If `value` has a [_falsy value_](https://developer.mozilla.org/en-US/docs/Glossary/Falsy), that includes 0, this variable would have the `innerValue` value instead, which i don't think is the right behaviour here, what do you think?

Reply:
In this case, it is not totally necessary because value is a string (it is parsed to int when the changeValue function is called). So, in the case the value is "0", it will be correctly evaluated using the `||` operator. Also, `value` cannot have any other falsy value (like " ", as it is not possible to write empty space on the number component). But I think it could be a good idea to change to `??` operator anyway.